### DISCUSSION TITLE
custom ntopng docker container startup

### DISCUSSION THREAD

**Discussion by TotieBash** [Security-Onion-Solutions/securityonion#8082] [Unsupported Versions] [answered]
I successfully added a new ntopng docker container.  I need help in figuring how to make it start the container last.  I need the so-redis to start first and then the new ntopng docker container.  Right now I manually do docker run for it.  Can someone point me on how SO starts the docker containers, I have a feeling it's manage be saltstack but not sure..  Thanks..

**Comment by dougburks** [comment, marked as answer]
We don't support third-party containers. Perhaps you could achieve your goal using software that is already included in Security Onion without needing to use ntopng.
